src/config/dynamic-theme-fixes.config

```diff
@@ -893,6 +894,9 @@ CSS
 .docs-text-ui-cursor-blink {
     fill: ${black} !important;
 }
+.ndfHFb-c4YZDc.ndfHFb-c4YZDc-i5oIFb .ndfHFb-c4YZDc-Bz112c:not([onclick]):not(:link):not(:visited) {
+ background-image: url('//ssl.gstatic.com/docs/common/viewer/v3/v-sprite26.svg')!important;
```

Comment:
4 Spaces
Space between `)` and `!important`

Reply:
Sorry for the trouble, didn't check this one.